plantower/plantower.py

```diff
@@ -134,3 +138,88 @@ def read(self, perform_flush=True):
                     return PlantowerReading(recv) # convert to reading object
             #If the character isn't what we are expecting loop until timeout
         raise PlantowerException("No message recieved")
+
+    def mode_change(self, mode=PMS_PASSIVE_MODE):
+        """
+            The default mode for the sensor is ACTIVE and whenever power OFF and ON (or sleep and wakeup)
+             the sensor starts in ACTIVE mode.
+            After mode changed, you need to wait some seconds for the sensor to get data again,
+            immediate reading sensor data gives you 0 values.
+        """
+
+        pms_cmd_changemode_passive = b'\x42\x4d\xe1\x00\x00\x01\x70'
```

Comment:
I would suggest you put these two variables (pms_cmd_changemode_passive and pms_cmd_changemode_active) as global variable for consistency.